proto/campaign/events.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+package tendermint.spn.campaign;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "campaign/campaign.proto";
+import "campaign/mainnet_account.proto";
+import "campaign/mainnet_vesting_account.proto";
+
+option go_package = "github.com/tendermint/spn/x/campaign/types";
+
+message EventCampaignCreated {
+  Campaign campaign = 1 [(gogoproto.nullable) = false];
+}
+
+message EventCampaignUpdated {
+  Campaign campaign = 1 [(gogoproto.nullable) = false];
+}
+
+message EventMainnetInitialized {
+  uint64 campaignID = 1;
+  uint64 mainnetID = 2;
+}
+
+message EventMainnetAccountCreated {
+  MainnetAccount account = 1 [(gogoproto.nullable) = false];
+}
+
+message EventMainnetAccountUpdated {
+  MainnetAccount account = 1 [(gogoproto.nullable) = false];
+}
+
+message EventMainnetAccountRemoved{
+  uint64 campaignID = 1;
```

Comment:
Why do we have the campaignID for this event but not for others?

Reply:
the `MainnetAccount` and `MainnetVestingAccount` types already include `campaignID`